.golangci.yml

```diff
@@ -18,6 +18,10 @@ linters:
 
 linters-settings:
   gci:
-    local-prefixes: github.com/suborbital
+    sections:
+      - standard
+      - default
+      - prefix(github.com/suborbital)
```

Comment:
now comes the better part, do we want to separate this to further?

```
- prefix(github.com/suborbital)
- prefix(github.com/suborbital/subo)
```

Reply:
We'll bike shed this tomorrow! I think I'm okay with three sections.